sample/HttpTrigger-Powershell/run.ps1

```diff
@@ -1,10 +1,10 @@
-if ($Env:req_query_name) 
```

Comment:
No, $Env will not work any longer.  The environment variables are now passed in as variables into the runspace that the PowerShell instance runs in.  As such, all environment variables will be treated as variables, e.g. $Env:req_query_name => $req_query_name, with the exception of $Env:input, because $input is a reserved variable in PowerShell. As such, the default input parameter will be $inputdata.

In the event that the user uses "input", then an exception will be thrown. See,

src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

and accompanying tests,

test/WebJobs.Script.Tests/TestScripts/PowershellFunctionGenerationTests.cs
- GenerateQueueTriggerFunction
- GenerateQueueTriggerFunction_WithInvalidInputName_Fails